<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NovelPropertiesDialog</class>
 <widget class="QDialog" name="NovelPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Game</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Title:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="titleEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Screen Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="widthCombo">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>640</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>800</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1024</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1440</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1680</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="heightCombo">
           <property name="editable">
            <bool>true</bool>
           </property>
           <item>
            <property name="text">
             <string>480</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>600</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>768</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>990</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1200</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>1050</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Font:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="fontSizeSpinner">
           <property name="toolTip">
            <string>Font size in pixels</string>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ChooseFontWidget" name="fontFamilyChooser">
           <property name="toolTip">
            <string>Font family</string>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="textSpeedLabel">
           <property name="text">
            <string>Text Speed:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="textSpeedSlider">
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="textSpeedValueLabel">
           <property name="minimumSize">
            <size>
             <width>32</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Project</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Engine Directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="engineDirectoryEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="engineDirectoryButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="media.qrc">
             <normaloff>:/media/document_open.png</normaloff>:/media/document_open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBuiltinBrowser">
         <property name="text">
          <string>Use built-in browser</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="browserLabel">
           <property name="text">
            <string>Browser</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="LineEdit" name="browserEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="browserButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="media.qrc">
             <normaloff>:/media/document_open.png</normaloff>:/media/document_open.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ChooseFontWidget</class>
   <extends>QComboBox</extends>
   <header>choosefontwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="media.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NovelPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NovelPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBuiltinBrowser</sender>
   <signal>toggled(bool)</signal>
   <receiver>browserEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBuiltinBrowser</sender>
   <signal>toggled(bool)</signal>
   <receiver>browserButton</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
